core/src/test/java/io/confluent/kafka/schemaregistry/zookeeper/MasterElectorTest.java

```diff
@@ -46,7 +46,7 @@
 import static org.junit.Assert.fail;
 
 public class MasterElectorTest extends ClusterTestHarness {
-  private static final int ID_BATCH_SIZE =
+  static final int ID_BATCH_SIZE =
```

Comment:
Good catch! I had experimented with other tests that needed these to be package private, but forgot to change this back.